scheduler/task_finder.go

```diff
@@ -12,14 +12,11 @@ type TaskFinder interface {
 	FindRunnableTasks() ([]task.Task, error)
 }
 
-// DBTaskFinder fetches tasks from the database. Implements TaskFinder.
-type DBTaskFinder struct{}
-
-// FindRunnableTasks finds all tasks that are ready to be run.
-// This works by fetching all undispatched tasks from the database,
-// and filtering out any whose dependencies are not met.
-func (self *DBTaskFinder) FindRunnableTasks() ([]task.Task, error) {
+// The old DBTaskFinder, with the dependency check implemented in Go, instead
+// of using $graphLookup
+type LegacyDBTaskFinder struct{}
 
+func (self *LegacyDBTaskFinder) FindRunnableTasks() ([]task.Task, error) {
```

Comment:
we can change self to "f" or some other more go-ish idom when we come across it in code that we're editing.